grade-scripts/dump-requests.js

```diff
@@ -35,13 +35,24 @@ const handleRequest = (requests, siteToCheck, request) => {
 
     requests.push([url, type])
 
-    let upgradedUrl = https.getUpgradedUrl(url)
+    let upgradedUrl
+    let tracker
 
-    if (url !== upgradedUrl) {
+    try {
+        upgradedUrl = https.getUpgradedUrl(url)
+    } catch (e) {
+        console.log(`error getting HTTPS upgraded URL for ${url}: ${e.message}`)
+    }
+
+    if (upgradedUrl && url !== upgradedUrl) {
         url = upgradedUrl
     }
 
-    let tracker = trackers.isTracker(url, siteToCheck, type)
+    try {
+        tracker = trackers.isTracker(url, siteToCheck, type)
+    } catch (e) {
+        console.log(`error getting tracker for ${url}: ${e.message}`)
+    }
```

Comment:
This will make it easier to catch bugs with privacy grade code.

Reply:
👍